oat-sa/tao-core
# feature/TR-918/support-rtl-inline-interaction

**Related to:** [TR-918](https://oat-sa.atlassian.net/browse/TR-918)

**Description:**

Adapt Inline text interaction to the item’s writing direction.

**Changes:**

- Link to the package and bundle files

**Extensions related** 
- https://github.com/oat-sa/extension-tao-itemqti/pull/1958

**Companion PR** 
- https://github.com/oat-sa/tao-core-ui-fe/pull/390

**How to check:**

- Create an Inline block interaction with Inline choice interaction item on RTL language. And duplicate it on LTR language.
- Create a test with both items and create TT with RTL language. Or use the test.zip on the ticket.
- Review test with the Guest user (LTR) and RTL TT.
- Check the drop-down is in the RTL direction when the Test Taker language is on RTL language.

**TAE Environment:**

- Link to the TAE on the ticket

**Local test:**

- There is a composer link on the ticket to install it locally
